apiclient/harvester_api/managers/volumesnapshots.py

```diff
@@ -2,12 +2,13 @@
 
 
 class VolumeSnapshotManager(BaseManager):
-    PATH_fmt = "v1/harvester/snapshot.storage.k8s.io.volumesnapshots/{ns}/{uid}"
+    _qs = "?filter=metadata.ownerReferences.kind=PersistentVolumeClaim"
+    PATH_fmt = "v1/harvester/snapshot.storage.k8s.io.volumesnapshots/{ns}/{uid}{qs}"
 
     def delete(self, name, namespace=DEFAULT_NAMESPACE, *, raw=False):
         path = self.PATH_fmt.format(uid=name, ns=namespace)
         return self._delete(path, raw=raw)
 
     def get(self, name="", namespace=DEFAULT_NAMESPACE, *, raw=False):
-        path = self.PATH_fmt.format(uid=name, ns=namespace)
+        path = self.PATH_fmt.format(uid=name, ns=namespace, qs=self._qs)
```

Comment:
> Sall we make `kind` filtering more flexible instead of hardcoded `qs`?
> Thus handy for us to handle other kinds of volume snapshots in the future.
> 

No, a **Manager** is designed to reflect a page in harvester dashboard, and we already have `VirtualMachineSnapshotManager` for VM snapshots and `BackupManager` for VM Backups.

The filter is implemented according to https://github.com/harvester/dashboard/pull/428, you can check the origin issue https://github.com/harvester/harvester/issues/2296 to get the full context.

if user want to list snapshots of VM, they should do `api.vm_snapshots.get()` rather than `api.vol_snapshots.get()`.

same as the VM backup, as we don't have page to let user do *volume backup*, so the `api.backups` is the endpoint to get VM backups.